lib/tasks/relevancy.rake

```diff
@@ -48,8 +48,6 @@ namespace :relevancy do
   task :send_ga_data_to_graphite do
     puts "Sending overall CTR to graphite"
     report_overall_ctr
-    puts "Sending viewcounts to graphite"
-    report_popular_queries
```

Comment:
Should this method still be called?  If not, can it be deleted?

Reply:
The `report_popular_queries` method is still used by the `relevancy:show_top_queries` task. Should we delete that task?

Reply:
Ah, it's still in use.  No, that's fine.